src/sql/workbench/services/objectExplorer/browser/dragAndDropController.ts

```diff
@@ -102,18 +102,29 @@ export class ServerTreeDragAndDrop implements IDragAndDrop {
 			let providerName = this.getProviderNameFromElement(element);
 			if (providerName === 'KUSTO') {
 				finalString = element.nodeTypeId !== 'Function' && escapedName.indexOf(' ') > 0 ? `[@"${escapedName}"]` : escapedName;
-			} else {
+			} else if (providerName === mssqlProviderName) {
 				finalString = escapedSchema ? `[${escapedSchema}].[${escapedName}]` : `[${escapedName}]`;
+			} else if (providerName === 'PGSQL') {
```

Comment:
Let's make a constant for this as well and put it in sql/platform/connection/common/constants